proto/frequenz/api/common/v1/metrics.proto

```diff
@@ -147,7 +66,7 @@ message AggregatedMetricSample {
 // `AggregatedMetricSample`, but not both simultaneously. Setting one will
 // nullify the other.
 message MetricSampleVariant {
-  oneof metric_return_type {
+  oneof metric_sample_type {
```

Comment:
`metric_return_type` sounded odd. How does `metric_sample_type` sound?

Reply:
lol I never fixed that one. much better! it was just a placeholder as I was missing a better idea at the time!